src/Router/BaseRouter.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * Implements BaseRouter, a Router to handle the base of a LORIS
+ * install.
+ *
+ * PHP Version 7
+ *
+ * @category Router
+ * @package  Router
+ * @author   Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\Router;
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Message\ResponseInterface;
+use \Psr\Http\Server\RequestHandlerInterface;
+use \LORIS\Http\StringStream;
+
+/**
+ * Handles the root of a LORIS install. It will mostly delegate to the
+ * A BaseRouter handles the base of a LORIS install and delegates to
+ * other routers (such as Module routers)
```

Comment:
Is there something missing here? The end of line 22 does continue well with start of line 23. Need some syntax/grammatical fix

Reply:
yes, this comment needs to be fixed. I think I was in too much of a rush while fixing phpcs complaints about comments

Reply:
Should be fixed now.